lib/encoder.rb

```diff
@@ -0,0 +1,16 @@
+require './lib/crypto_tool'
+
+class Encoder < CryptoTool
+
+  def random_key
+    rand.to_s[2..6]
```

Comment:
`rand` generates a random float between 0 and 1, it MIGHT be possible to generate one with less than 6 digits